modules/apps/29-fee/keeper/keeper.go

```diff
@@ -97,9 +107,9 @@ func (k Keeper) IsFeeEnabled(ctx sdk.Context, portID, channelID string) bool {
 
 // SetCounterpartyAddress maps the destination chain relayer address to the source relayer address
 // The receiving chain must store the mapping from: address -> counterpartyAddress for the given channel
-func (k Keeper) SetCounterpartyAddress(ctx sdk.Context, address, counterpartyAddress string) {
+func (k Keeper) SetCounterpartyAddress(ctx sdk.Context, address string, counterpartyAddress sdk.AccAddress) {
```

Comment:
This should remain a string. You don't know that the counterpartyAddress will be an sdk.AccAddress.

All you know is that address is `sdk.AccAddress`